doc/en/deprecations.rst

```diff
@@ -80,22 +80,35 @@ Note that ``from_parent`` should only be called with keyword arguments for the p
 
 .. deprecated:: 5.2
 
-The default value of ``junit_family`` option will change to ``xunit2`` in pytest 6.0, given
-that this is the version supported by default in modern tools that manipulate this type of file.
+The default value of ``junit_family`` option will change to ``xunit2`` in pytest 6.0, which
+is an update of the old ``xunit1`` format and is supported by default in modern tools
+that manipulate this type of file (for example, Jenkins, Azure Pipelines, etc.).
 
-In order to smooth the transition, pytest will issue a warning in case the ``--junitxml`` option
-is given in the command line but ``junit_family`` is not explicitly configured in ``pytest.ini``::
+Users are recommended to try the new ``xunit2`` format and see if their tooling that consumes the JUnit
+XML file supports it.
 
-    PytestDeprecationWarning: The 'junit_family' default value will change to 'xunit2' in pytest 6.0.
-      Add 'junit_family=legacy' to your pytest.ini file to silence this warning and make your suite compatible.
+To use the new format, update your ``pytest.ini``:
```

Comment:
That suggests that by not adding `junit_family=xunit2` the modern format will not be used. That's not true IIUC, is it?

Reply:
It is True as per pytest <6.0, only in 6.0 the default will be `xunit2`.

Reply:
Maybe mentioning this fact may be helpful, e.g.
```suggestion
To use the new format in pytest <6.0, update your ``pytest.ini``:
```

Reply:
Good point, thanks!